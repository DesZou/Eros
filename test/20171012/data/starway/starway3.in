10000 30000 400
9469 20422
6470 2281
1187 27397
3967 22332
4204 13763
1292 8654
2851 27089
1161 25530
4607 2389
9879 22665
1899 21130
6851 23297
9424 24974
5024 1009
6599 3266
5412 6949
885 27423
8764 17598
9621 15352
7775 7624
2828 13410
2614 26455
4801 15239
2015 10783
9433 863
9702 12215
8137 3904
1780 10213
7835 26480
8144 23429
7639 13062
1158 8122
2976 6805
1906 9069
9037 5785
2991 29261
8224 22333
4105 17907
2272 3052
4893 23625
3706 22126
5219 20577
2280 2881
6898 28393
5556 18594
4152 281
5510 4764
7078 23588
6792 19798
8853 17939
7834 16576
2456 25629
1467 19749
9802 13214
2239 7666
2168 16781
546 1705
9857 3694
8441 22043
4370 8627
3629 11577
3025 5274
9781 20432
5328 3225
4182 13663
2801 19052
6099 11118
1485 11977
7632 4899
6163 19619
2020 10270
9503 3381
2414 3615
9352 13612
9426 21320
1333 13641
7819 18011
5858 9907
3402 8794
1160 350
3107 25743
900 24831
6821 7541
2453 29595
9009 2234
9201 17794
9452 9951
4343 24045
9357 28187
4682 8441
3958 26624
368 21306
9034 3189
3217 17563
828 28718
520 22526
8414 26859
4884 9036
3415 3297
333 3450
6354 11441
5779 13955
2060 21992
7628 9450
7503 27443
5989 2441
3272 18156
3415 11919
4838 4273
6918 8949
477 20613
7697 19944
8003 29611
5001 24933
7267 5251
9388 18644
1257 24104
8659 16453
7972 27701
6901 12064
7201 24752
1783 4450
616 7890
7674 6099
2316 6032
2015 292
9983 23547
4147 29379
1813 5356
7929 23528
222 20930
3496 3818
6282 5349
3635 2223
9752 3183
4920 7062
4055 5567
8457 362
6115 6421
733 13893
618 21883
7896 10759
3787 22070
9585 11124
8423 5711
7522 13268
4125 23105
3320 22305
1933 18796
4521 8668
3981 18590
8079 13286
2112 12194
9484 5599
1848 13213
2786 6599
3371 18298
7300 9431
6181 21376
3286 29293
1567 28963
7238 14563
4325 2125
9759 16139
334 16704
1907 8869
8944 22228
4097 7818
2181 7458
5895 24715
226 29184
9274 7198
73 8441
4659 7173
9809 22671
6713 8672
5008 24713
8386 3720
3529 1729
7237 25966
183 4067
7426 25187
4911 8944
3925 9404
6928 11886
715 15536
4940 25924
8925 29183
697 4710
735 29130
6545 11050
3138 28823
5535 28024
6401 29416
7794 23548
1999 15944
6221 6594
9394 14927
5658 10402
5116 15860
5143 4257
2401 7095
6219 11387
9112 21162
5881 13995
4957 2321
8010 8431
9342 5864
5845 6509
9549 5727
6908 19393
3534 12515
4407 24077
5757 14682
689 22506
1652 20320
4372 22673
6710 8474
6187 27681
1149 8888
7210 19561
201 17284
5424 4729
1926 10419
7440 27649
2034 25473
6789 5428
9450 16627
1056 8673
6654 4440
3040 6893
9165 20582
2826 29771
5793 4732
7670 2682
8402 9270
5768 12264
6548 4918
3274 9228
6025 4914
4112 11471
8872 2808
9528 1620
3241 13208
5172 25400
1976 28549
1473 8482
3352 27629
3993 17399
3615 19808
17 16848
7070 9209
8378 21671
3645 6697
3233 16195
105 18467
7381 16157
2870 11869
1205 6593
115 12207
9502 5267
8816 22082
4947 17433
8969 20250
1537 17553
8904 19252
2785 18017
4380 4664
9843 20044
4035 12996
1642 24586
3337 13097
3656 29203
3055 23437
9664 29487
2063 1209
4184 24582
4410 514
2169 13952
8452 10323
5426 5942
8047 13663
6766 3986
3695 25978
2254 26457
4974 6461
7937 22975
9823 21257
941 7551
3059 29194
7780 7919
729 18330
5301 26524
8185 2681
4621 14490
388 12339
3533 19287
9459 16660
5231 4606
8032 13829
495 12303
3939 7690
3204 3948
894 23277
3189 8948
3738 26053
119 28336
1768 28828
3756 14092
8473 12675
5782 11884
3188 4717
3969 16599
916 11606
3348 8343
1261 1070
3104 13644
8199 27169
7077 9891
6213 18705
8553 21276
8538 26058
7696 18790
2720 29118
9355 12141
6551 5721
6326 8660
1527 23538
2576 9490
1517 1268
7450 15934
1196 15602
2320 8244
2577 8910
433 16566
134 18127
1982 10065
2414 24107
6907 7111
1862 20428
980 9470
2262 25195
5717 11480
321 12673
9908 25220
3915 14506
8276 12722
8910 4188
7129 125
2951 22229
1800 16083
874 24172
7859 13690
8175 27697
8201 4656
8327 16866
8166 6222
8879 20749
7587 24637
3380 8229
5473 5659
6685 10341
898 13205
2877 12781
7390 3676
4063 27006
224 11283
8882 23452
2476 4127
5317 27259
4443 28050
6411 11383
3612 16089
1478 15045
2406 26908
4618 17044
4263 13245
6521 11828
5320 11973
8061 21215
6885 25861
452 15165
3296 23772
4885 16438
1442 5603
685 27347
1089 20183
4270 1125
4465 9267
970 27769
1408 25319
7556 14364
2613 1345
7021 24804
3726 8362
1042 18500
3845 20174
1162 25924
5101 16760
7629 2467
