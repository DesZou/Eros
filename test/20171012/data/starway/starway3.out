653.172259056
